cocoa/src/toga_cocoa/widgets/textinput.py

```diff
@@ -119,5 +119,6 @@ def is_valid(self):
         # Cocoa TextInput can't ever be in an invalid state, so it is always valid
         return True
 
+    # Not sure on how to alert that on_confirm is not implimented, guidance needed.
```

Comment:
Ok - so I don't have any examples of this in code that I can point you at, but the approach will be to put an attribute on the implementation classes that *don't* support `on_confirm`, and check for the existence of that attribute on the `setter` in the core. I'll add details on a comment on the setter.